packages/protocol/lib/test-utils.ts

```diff
@@ -418,3 +423,113 @@ export async function addressMinedLatestBlock(address: string) {
+		'GovernanceApproverMultiSig',
+		artifacts
+	)
+	const registry: RegistryInstance = await getDeployedProxiedContract('Registry', artifacts)
+	// @ts-ignore
+	await lockedGold.lock({ value: '10000000000000000000000000' })
+	const proposalTransactions = await Promise.all(
+		contractsToOwn.map(async (x: string) => {
+			return {
+				value: 0,
+				destination: (await getDeployedProxiedContract(x, artifacts)).address,
+				data: Buffer.from(
+					stripHexEncoding(
+						// Any contract's `transferOwnership` function will work here as the function signatures are all the same.
+						// @ts-ignore
+						registry.contract.methods.transferOwnership(to).encodeABI()
+					),
+					'hex'
+				),
+			}
+		})
+	)
+	await governance.propose(
+		proposalTransactions.map((x: any) => x.value),
+		proposalTransactions.map((x: any) => x.destination),
+		// @ts-ignore
+		Buffer.concat(proposalTransactions.map((x: any) => x.data)),
+		proposalTransactions.map((x: any) => x.data.length),
+		'URL',
+		// @ts-ignore: TODO(mcortesi) fix typings for TransactionDetails
+		{ value: web3.utils.toWei(config.governance.minDeposit.toString(), 'ether') }
+	)
+
+	await governance.upvote(proposalId, 0, 0)
+	await timeTravel(config.governance.dequeueFrequency, web3)
+	// @ts-ignore
+	const txData = governance.contract.methods.approve(proposalId, dequeuedIndex).encodeABI()
+	await multiSig.submitTransaction(governance.address, 0, txData)
+	await timeTravel(config.governance.approvalStageDuration, web3)
+	await governance.vote(proposalId, dequeuedIndex, VoteValue.Yes)
+	await timeTravel(config.governance.referendumStageDuration, web3)
+	await governance.execute(proposalId, dequeuedIndex)
+}
+
+/*
+ * Helpers for verification
+ */
+export enum KeyOffsets {
+  VALIDATING_KEY_OFFSET,
+  ATTESTING_KEY_OFFSET,
+  NEW_VALIDATING_KEY_OFFSET,
+  VOTING_KEY_OFFSET,
+}
+
+// Private keys of each of the 10 miners, in the same order as their addresses in 'accounts'.
+export const accountPrivateKeys: string[] = [
+  '0xf2f48ee19680706196e2e339e5da3491186e0c4c5030670656b0e0164837257d',
+  '0x5d862464fe9303452126c8bc94274b8c5f9874cbd219789b3eb2128075a76f72',
+  '0xdf02719c4df8b9b8ac7f551fcb5d9ef48fa27eef7a66453879f4d8fdc6e78fb1',
+  '0xff12e391b79415e941a94de3bf3a9aee577aed0731e297d5cfa0b8a1e02fa1d0',
+  '0x752dd9cf65e68cfaba7d60225cbdbc1f4729dd5e5507def72815ed0d8abc6249',
+  '0xefb595a0178eb79a8df953f87c5148402a224cdf725e88c0146727c6aceadccd',
+  '0x83c6d2cc5ddcf9711a6d59b417dc20eb48afd58d45290099e5987e3d768f328f',
+  '0xbb2d3f7c9583780a7d3904a2f55d792707c345f21de1bacb2d389934d82796b2',
+  '0xb2fd4d29c1390b71b8795ae81196bfd60293adf99f9d32a0aff06288fcdac55f',
+  '0x23cb7121166b9a2f93ae0b7c05bde02eae50d64449b2cbb42bc84e9d38d6cc89',
+]
+
+export async function getVerificationCodeSignature(
+  account: string,
+  issuer: string,
+  identifier: string,
+  accounts: string[]
+): Promise<[number, string, string]> {
```

Comment:
nit: return named parameters instead of requiring consumer to know ordering [v,r,s]